universal-login-react/src/ui/TopUp/Fiat/CountryDropdown.tsx

```diff
@@ -19,7 +19,8 @@ export const CountryDropdown = ({selectedCountry, setCountry, setCurrency}: Coun
 
   const countrySelectionButton = () => {
     const country = countries.find(({name}) => name === selectedCountry);
-    const {name, flag} = country || {} as Partial<Country>;
+    const {name, code} = country || {} as Partial<Country>;
+    const flag = code && require(`../../assets/flags/${code.toLowerCase()}.svg`);
```

Comment:
I would extract `require(`../../assets/flags/${code.toLowerCase()}.svg`)` to for example `getFlagSvg(code)`

Reply:
Proposed solution:
```ts
getFlagPath(code.toLowerCase());
...
const getFlag = (code) => {
  return getAsset(`${code.toLowerCase()}.svg`);
}

const getAsset = (path) => {
  return require(`../../assets/flags/${path}`);
}
```